sympy/physics/control/lti.py

```diff
@@ -192,7 +245,7 @@ def num(self):
         p*s + s**2 + 3
         >>> G2 = TransferFunction((p + 5)*(p - 3), (p - 3)*(p + 1), p)
         >>> G2.num
-        (p - 3)*(p + 5)
+        p**2 + 2*p - 15
```

Comment:
Well, that was also my first idea back then, but what if the user wants the unexpanded form? Expansion of `num` and `den` is just one step away. Consider this simple example:
```python
>>> from sympy import *
>>> from sympy.abc import *
>>> a = x**2 + x - 2
>>> b = y - 4
>>> a*b
(y - 4)*(x**2 + x - 2)  # no automatic expansion
```

Reply:
In my opinion, having a definite form would enable us to compare two `TransferFunction` instances in later phases. Moreover, Transfer functions' numerators and denominators are always polynomials, so it should be better, in my opinion, to show them in their typical polynomial form (`a_n*x**n + a_(n-1)*x**(n-1)+...+a_0`). 
Unfortunately, I didn't find a single example in the docs of MATLAB and Mathematica where they used an expression whose `num` and `den` were not already in the polynomial form.

You can have a look-
https://www.mathworks.com/help/control/ref/tf.html
https://reference.wolfram.com/language/ref/TransferFunctionZeros.html
https://harold.readthedocs.io/en/latest/generated/harold.Transfer.html#harold.Transfer

In MATLAB and Harold, they have strictly implemented the polynomial form. Users have no freedom to instantiate a transfer function by inputting an expression. They can only input the coefficients of the polynomial. There is a form in MATLAB to create a transfer function instance with poles and zeros, but that also maintains its polynomial characteristic.

I'll do what you suggest as a lot of discussions have already been done.

Reply:
Okay, fair enough, let's go with the polynomial form!